libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/IndividualFeatureImportanceView/LargeIndividualFeatureImportanceView/LocalImportanceChart.tsx

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getTheme, Slider, Stack, Toggle } from "@fluentui/react";
+import {
+  BasicHighChart,
+  defaultModelAssessmentContext,
+  getPrimaryChartColor,
+  ILocalExplanations,
+  IsClassifier,
+  LoadingSpinner,
+  MissingParametersPlaceholder,
+  ModelAssessmentContext,
+  ModelTypes,
+  WeightVectorOption
+} from "@responsible-ai/core-ui";
+import { ClassImportanceWeights } from "@responsible-ai/interpret";
+import { localization } from "@responsible-ai/localization";
+import { Dictionary } from "lodash";
+import React from "react";
+
+import { localImportanceChartStyles } from "./LocalImportanceChart.styles";
+import { getSortedData } from "./localImportanceChartUtils";
+
+export interface ILocalImportanceChartProps {
+  rowNumber?: number;
+  data?: ILocalExplanations;
+  theme?: string;
+  isLocalExplanationsDataLoading?: boolean;
+  localExplanationsErrorMessage?: string;
+  selectedWeightVector: WeightVectorOption;
+  weightOptions: WeightVectorOption[];
+  weightLabels: Dictionary<string>;
+  modelType: ModelTypes;
+  onWeightChange: (option: WeightVectorOption) => void;
+}
+
+export interface ILocalImportanceChartState {
+  sortAbsolute: boolean;
+  sortedData: Array<{ [key: string]: number[] | number | undefined }>;
+  topK: number;
+}
+export interface ILocalImportanceData {
+  label: string;
+  value: number;
+}
+
+export const regressionKeyValue = "regressionScore";
```

Comment:
Do we want to move this to constants.ts?

Reply:
moved to constants